common/common.go

```diff
@@ -31,6 +31,7 @@ const (
 	MetadataServiceIP      = "169.254.169.255"
 	MetadataServicePubPort = 80
 	MetadataServicePrvPort = 2375
+	MetadataServiceRegSock = "/var/run/rkt-metadata-svc.sock"
```

Comment:
Also, it is typically customary to create a directory for yourself under /run, so /run/rkt/metadata-svc.sock